core/templates/dev/head/profile/Profile.js

```diff
@@ -29,22 +31,87 @@ oppia.controller('Profile', [
     $rootScope.loadingMessage = 'Loading';
     $http.get(profileDataUrl).success(function(data) {
       $rootScope.loadingMessage = '';
+      $scope.username = {
+        title: 'Username',
+        value: data.username,
+        helpText: (data.username)
```

Comment:
(If it's an issue with non-usernames, maybe we should only show it if the username is too long?)